oauthenticator/globus.py

```diff
@@ -195,7 +200,29 @@ def authenticate(self, handler, data=None):
                     'globus.org/app/account'
                     )
             )
-        return username
+        return {
+            'name': username,
+            'auth_state': {
+                'client_id': self.client_id,
+                'tokens': {
+                    tok: v for tok, v in tokens.by_resource_server.items()
+                    if tok not in self.exclude_tokens
+                },
+            }
+        }
```

Comment:
Did they change the return type, such that this passes the token to their sqlite DB?

Reply:
They did. Where it used to just be `username`, it's now a dict. Not long ago, I think only one or two authenticators were returning this dict. As of this morning, it looks like all of them are. I'll have to revisit their changes and amend the ones here.